routes/segments_routes.py

```
@@ -1,76 +1,190 @@
-)
-from modules.devide.segments import add_segments
-
-segments_bp = Blueprint('/segments', __name__)
-
-@segments_bp.route('/csv', methods=['GET'])
-def segments_csv():
+from flask import Blueprint, request, jsonify
+import os
+import io
+from datetime import datetime
+from flask import send_file, jsonify
```

Ruff (0.11.9)

Redefinition of unused `jsonify` from line 1

Remove definition: `jsonify`

(F811)